hadoop-ozone/dist/src/main/compose/test-all.sh

```diff
@@ -41,8 +46,13 @@ for test in $(find "$SCRIPT_DIR" -name test.sh | sort); do
       echo "ERROR: Test execution of $(dirname "$test") is FAILED!!!!"
   fi
   RESULT_DIR="$(dirname "$test")/result"
-  cp "$RESULT_DIR"/robot-*.xml "$RESULT_DIR"/docker-*.log "$RESULT_DIR"/*.out* "$ALL_RESULT_DIR"/
+  cp "$RESULT_DIR"/robot-*.xml "$RESULT_DIR"/docker-*.log "$RESULT_DIR"/*.out* "$RESULT_DIR"/*.exec "$ALL_RESULT_DIR"/
 done
 
 rebot -N "smoketests" -d "$SCRIPT_DIR/result" "$SCRIPT_DIR/result/robot-*.xml"
+if [ "$OZONE_WITH_COVERAGE" ]; then
+  cp /tmp/jacoco-combined.exec "$SCRIPT_DIR/result/"
```

Comment:
I don't see any `*.exec` files in [acceptance.zip](https://github.com/apache/hadoop-ozone/suites/812019480/artifacts/8877510) for the latest run.  Am I missing something?